_posts/2015-11-09-scrum-basics-2.md

```diff
@@ -0,0 +1,118 @@
+
+A typical Development Team consists of 7 members +/-2 with cross-functional abilities.
+They should organise themselves and decide on their own, how tickets will be implemented within a sprint.
+It’s also the Development Team's decision, how many tickets they decide to solve within a sprint.
+This freedom comes with the duty to commit completely to the goal to deliver all the planned tickets within a sprint.
+
+To being able to know how many tickets they are usually able to solve within a sprint, the team estimates all tickets by themselves.
+Nobody else is allowed to give estimates for tickets but the Development Team as they will solve those tickets in the end.
+During the sprint, the Development Team also has the duty to track their progress.
+That means, they keep the Sprint/Scrum Board clean and move the cards across the board according to their progress.
+Everyone in the team, including PO and SM, should always know the status of each ticket within a sprint.
+
+The most important task for the Development Team, also in Scrum, is to deliver.
+Just in Scrum they can tell themselves, how much time they need to deliver something, which gives them the freedom to achieve the big goal of the Scrum Framework: to deliver quality products.
+
+# Product Owner
+
+{% image blog-scrum2-productowner.jpg 30% left %}{% endimage %}
+
+The Product Owner is a very important role in Scrum.
+I’ve heard trainers say that a Scrum Team can rise or fall with their PO.
+The speciality in Scrum is, that the PO is not the boss of the Development Team, but part of the team.
+This is not easy, especially for POs who have worked as Project Managers in the past.
+In Scrum they are a lateral leader.
+
+However, they do decide what the team is going to develop next.
+They are setting up the Product Backlog and decide the business value of each ticket and with that the ticket order.
+That means, the PO is responsible for the Return of Investment (ROI).
+They decide if fixing a bug is more important than developing a new feature.
+They spend a lot of time maintaining the Product Backlog, which means creating new tickets and prioritising them as well as keeping the backlog up to date.
+Several techniques exist of how to prioritise a Backlog.
+The technique to be used mainly depends on the product, the team is working on.
+
+To create the tickets the PO is also responsible for gathering all the requirements as well as to talk to the stakeholders.
+Managing all the stakeholders for a product can be quite time consuming.
+But there are some techniques that can help the PO to keep an overview.
+
+The PO's superior task is to create a common product vision and to steer the product release.
+They are also responsible for accepting (or not ;) ) the developed tickets from the Development Team.
+As you can see, the Product Owner is a very important and complex role in Scrum and so it’s no wonder that several trainings exist one should participate in before agreeing on this job.
```

Comment:
As you can see, the Product Owner is a very important and complex role in Scrum and so it’s no wonder that several training sessions exist and that one should participate in before agreeing on this job.